x-pack/plugins/endpoint/package.json

```diff
@@ -6,7 +6,8 @@
   "license": "Elastic-License",
   "scripts": {},
   "dependencies": {
-    "react-redux": "^7.1.0"
+    "react-redux": "^7.1.0",
+    "redux-devtools-extension": "^2.13.8"
```

Comment:
I think this should be in `devDependencies` as its a dev tool?

Reply:
I'll try to get that workin.